CondCore/SiPixelPlugins/plugins/BuildFile.xml

```diff
@@ -62,3 +52,7 @@
   <use name="CalibTracker/SiPixelESProducers"/>
   <use name="boost_python"/>
 </library>
+
+<library file="plugin.cc" name="CondCoreSiPixelPluginsPlugin">
+  <use name="CondCore/SiPixelPlugins"/>
```

Comment:
```suggestion<use name="CalibTracker/StandaloneTrackerTopology"/>
<use name="CalibTracker/SiPixelESProducers"/>
<use name="CondCore/ESSources"/>
<use name="CondFormats/SiPixelObjects"/>
<use name="CondFormats/DataRecord"/>
```

Reply:
is `<flags EDM_PLUGIN="1"/>` implied in the `plugins` subdirectories ?

Reply:
It is. I checked that the library gets compiled as a plugin by inspecting the `$CMSSW_BASE/lib/slc7_amd64_gcc820` directory, and it contains `pluginCondCoreSiPixelPluginsPlugin.so`. Is it recommended to be explicit anyway?

Reply:
I think I also like it more explicit.

Reply:
In the[ compilation log ](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-20f9d2/8810/build.log) there is now:
```
***WARNING: Multiple usage of "CondCore/SiPixelPlugins". Please cleanup "use" in "non-export" section of "src/CondCore/SiPixelPlugins/plugins/BuildFile".
```

I think this should be solved if you apply the suggestion at https://github.com/cms-sw/cmssw/pull/31165#discussion_r472426576

Reply:
Thanks for pointing that out @mmusich. By the way, now that you are here as the developer of that package, is there a reason why all the files get compiled into individual libraries? They share almost all of their dependencies anyway. Couldn't we just put them all into the same plugins library? BuildFiles that only build one target are also much easier to check with automated tools.

Reply:
@guitargeek, yes there is a reason. They need to be compiled as separate libraries in order for the [Payload Inspector ](https://cms-conddb.cern.ch/cmsDbBrowser/payload_inspector/Prod) CondDB backend code to be able to "discover" each plugin separately. 
AFAIU what you propose would destroy such functionality. @ggovi FYI.

Reply:
I see, good to know. Then I'll just remove the duplicate `use`.